hibernate-core/src/test/resources/hibernate.properties

```diff
@@ -24,4 +30,4 @@ hibernate.cache.region.factory_class org.hibernate.testing.cache.CachingRegionFa
 javax.persistence.validation.mode=NONE
 hibernate.service.allow_crawling=false
 hibernate.session.events.log=true
-hibernate.hql.bulk_id_strategy.global_temporary.drop_tables=true
\ No newline at end of file
+hibernate.hql.bulk_id_strategy.global_temporary.drop_tables=true
```

Comment:
Did you mean to commit the changes to this file? Seems like you're writing a custom config file in the test harness.